### PR TITLE
Lab9

### PR DESCRIPTION
**Pull request by afonyaa**

### CODE REVIEW

**Comment by none-word** [comment]
Good job!

File: k8s/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,19 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: server-deployment
+spec:
+  replicas: 1
```
Comment (AKurmazov) [inline k8s/deployment.yaml:6]: nit: There had to be at least 3 replicas

**Comment by AKurmazov** [comment]
Well done 👍

**Comment by Maxkoz777** [comment]
Approved)

**Pull request closed without merging**